### ISSUE TITLE
Education, Entertainment, Shopping and Work

### ISSUE THREAD

**Issue by aubergine10** [Celisuis/AdvancedDistrictOptions#3]
Restricting these things will likely have major impact or conflict with the popular Real Time mod by @dymanoid.

For example, in Real Time, citizens get daily schedule that might involve shopping on their way home from work or in lunch break, etc.

Maybe disable these options if Real Time mod is found (workshop id: 1420955187)

Note: In respect to #2, the education, entertainment, shopping and work make sense to be macroscopic.

**Comment by Celisuis** [comment]
Yup. I'll add a check in for Real Time, Ill separate the services as well to make them individual, and keep these as district-wide

**Comment by aubergine10** [comment]
IMO the "cim activities" stuff might be better separated in to separate mod - it's a very different kettle of fish to services stuff.

Likewise, the industry stuff is also possibly worthy of separate mod as it too has it's own distinct use case, not to mention the likely issues with 'not enough goods', 'not enough customers', etc.

**Comment by Celisuis** [comment]
Very true. I'll see what it's like when I come to rewrite them. It won't be any more time consuming since most of the code is done

**Comment by aubergine10** [comment]
It's not so much the code that's the issue, more the game mechanics. I remember when Dymanoid was implementing cim activity management in to Real Time, and similarly the work schedules stuff with industry in particular, and it caused all kinds of notification bubbles because places weren't being serviced or visited enough. I suspect it will require a lot more effort than simply restricting where cims can go to work, shop, etc.

Similarly, with industry, I think it needs a completely different approach that's more based around supply chains rather than limiting by district. For example, a factory would be limited to getting supplies from warehouse 1 and 2, with fallback supply from outside connection 4. That sort of thing.

**Comment by Celisuis** [comment]
I get what you mean. I can definitely look into that. It's definitely possible as that was the way I originally had it planned, I just didn't flesh it out enough before going this route.

I do like the idea of limiting to building though.
